src/consts.ts

```diff
@@ -44,6 +52,7 @@ export const fiveZeroThreeConfig = {
 
 export const bundledConfig = [
   fiveOneThreeConfig,
+  fiveTwoZeroConfig,
```

Comment:
I have not added `fiveTwoZeroConfig` to the top so that default still remains 5.1.3. We can make it default when mainnet goes live